src/run_job_mix.sh

```diff
@@ -0,0 +1,425 @@
+#!/bin/bash -e
```

Comment:
Why is this file duplicated?

Reply:
Oh my bad, removing. Had it there for testing purposes.